pkg/client/result.go

```diff
@@ -14,6 +16,31 @@ type Result struct {
 	Rows    []Row    `json:"rows"`
 }
 
+// Due to big int number limitations in javascript, numbers should be encoded
+// as strings so they could be properly loaded on the frontend.
+func (res *Result) PrepareBigints() {
+	for i, row := range res.Rows {
+		for j, col := range row {
+			if col == nil {
+				continue
+			}
+
+			switch reflect.TypeOf(col).Kind() {
+			case reflect.Int64:
+				val := col.(int64)
+				if val < -9007199254740991 || val > 9007199254740991 {
+					res.Rows[i][j] = strconv.FormatInt(col.(int64), 10)
+				}
+			case reflect.Float64:
+				val := col.(float64)
+				if val < -999999999999999 || val > 999999999999999 {
+					res.Rows[i][j] = strconv.FormatFloat(val, 'e', -1, 64)
+				}
```

Comment:
I understand that, but while i was testing int64/float64 queries i noticed that when i fetch table rows without a query (basically a SELECT * FROM table query) rows for real/double precision columns identify as float64 in go, but when i ran a custom query, like `SELECT val FROM table`, the column type was set to string. While you're saying that float64 conversion is not necessary, it is in fact necessary to correctly format the value in order to properly render it in javascript. Otherwise, formatting float64 values via go's json will cause the same issue as with int64 values. Maybe im missing something, but without above mentioned conversions i could not get the numbers to show up correctly on the frontend.